index.html

```diff
@@ -162,25 +162,39 @@ <h2 class="salutation">Tonic</h2>
             <li><a href="https://twitter.com/home"><img src="images/Vector (2).jpg" alt="twitter"></a></li>
         </ul> 
     </section>  
+    <div>
+        <button class="get-resume" type="button">Get my Resume</button>
 
-            
-</div>
-<button class="get-resume" type="button">Get my Resume</button>
-</div>
-<section class="skills">
-
-<ul class="about-skill">
-    <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt="">
-    </li> 
-    <p><img src="images/js.jpg" alt="">JavaScript</p>
-    <p><img src="images/html.jpg" alt="">HTML</p>
-    <p><img src="images/ccs3.jpg" alt="">CSS</p>
+    </div>
     
-    <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-    <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
-    <hr>
-</ul>
+    <section class="skills">
 
-    
-</section>
\ No newline at end of file
+        <ul class="about-skill">
+            <li class="aboutl">Languages <img src="images/arrowDown.jpg" alt=""></li> 
+            <p><img src="images/js.jpg" alt="">JavaScript</p>
+            <p><img src="images/html.jpg" alt="">HTML</p>
+            <p><img src="images/ccs3.jpg" alt="">CSS</p>
+            
+            <li class="about">Frameworks <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+            <li class="about">Skills <img src="images/arrowRight.jpg" alt=""></li> 
+            <hr>
+        </ul>
+    </section>
+</section>
+
+<!-- This section is about contact form -->
+<section class="contact-form">
+    <h2 class="contact-me ">Contact me</h2>
+    <p class="contact-item">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. 
+        I’d love to help with it!</p>
+        <form action="#" class="form contact-item">
+            <input class="form-item" type="text" maxlength="30" placeholder="Lorem Ipsum" required>
+            <input class="form-item" type="email" placeholder="loremipsum@gmail.com" required>
+            <textarea class="form-item" name="comment" id="comment" cols="50" rows="10" placeholder="Write your message here..."></textarea>
+            <button class="form-item" type="submit">Get in touch</button>
```

Comment:
- [ ] Good job with button interactions 👍👍 Please consider fixing the `hover` and `active` interactions for contact form's button according to the project requirements

> _Implement the button interactions (enable, hover, etc.)._


Your hover interaction | Given design
---| ---|
![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/27c854f4-cd3b-4611-8d81-130907c5ad22) | ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/b2af8a5d-8952-4360-ab12-d8f638a6c6a2)
 
 Your active interaction | Given design
---| ---|
![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/50a9f818-16dc-4c98-9185-a85c4a878678) | ![image](https://github.com/ausiemass/portfolio-austin/assets/106119105/a9640bc0-b02e-4204-b13c-6aa461614e16)